_styles/-theme.scss

```diff
@@ -9,18 +9,18 @@
   --background: #ffffff;
   --background-alt: #fafafa;
   --light-gray: #e0e0e0;
-  --gray: #808080;
+  --gray: none;
```

Comment:
Why did this have to become none?

Reply:
As we discussed, revert this change and make a more specific edit in `header.scss` if needed.